platform/android/java/lib/src/org/godotengine/godot/Godot.kt

```diff
@@ -823,8 +823,17 @@ class Godot(private val context: Context) {
 	 * Returns true if `Vulkan` is used for rendering.
 	 */
 	private fun usesVulkan(): Boolean {
-		val renderer = GodotLib.getGlobal("rendering/renderer/rendering_method")
-		val renderingDevice = GodotLib.getGlobal("rendering/rendering_device/driver")
+		var renderer = GodotLib.getGlobal("rendering/renderer/rendering_method")
+		var renderingDevice = GodotLib.getGlobal("rendering/rendering_device/driver")
+		val cmdline = getCommandLine()
+		var index = cmdline.indexOf("--rendering-method")
+		if (index > -1 && cmdline.size > index + 1) {
+			renderer = cmdline.get(index + 1)
+		}
+		index = cmdline.indexOf("--rendering-driver")
+		if (index > -1 && cmdline.size > index + 1) {
+			renderingDevice = cmdline.get(index + 1)
+		}
```

Comment:
Can you add a logging statement for the `renderer` and `renderingDevice`.

Reply:
```
01-02 18:13:22.389 28468 28468 D Godot   : usesVulkan()
01-02 18:13:22.389 28468 28468 D Godot   : 			renderingDevice: vulkan (ProjectSettings)
01-02 18:13:22.389 28468 28468 D Godot   : 			renderer: gl_compatibility (CommandLine)
01-02 18:13:22.389 28468 28468 D Godot   : 	
```